lib/app.js

```diff
@@ -136,7 +136,7 @@ const MAPPINGS = {
   screenshots: {
     path: ['ds:5', 1, 2, 78, 0],
     fun: (screenshots) => {
-      if (screenshots === null) return [];
+      if (!screenshots?.length) return [];
```

Comment:
is this different from?
```suggestion
      if (!screenshots) return [];
```

Reply:
Yes, it's different. If you remove `?.length`, then this if goes falsy and code on the line 140 will be executed, but when screenshots is empty, then map returns an empty array.
So, if you leave part `?.length`, then this check can be a little quicker.